#! /bin/sh /usr/share/dpatch/dpatch-run
## 08-init-fail.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix some failures in init script.

@DPATCH@
diff -urNad virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in.orig virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in
--- virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in.orig	2008-05-01 18:37:12.000000000 +0200
+++ virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in	2008-05-01 18:38:51.000000000 +0200
@@ -49,6 +49,7 @@
 if [ -z "$NOLSB" ]; then
     . /lib/lsb/init-functions
     fail_msg() {
+        [ -f "$VARFILE" ] && rm $VARFILE
         echo ""
         log_failure_msg "$1"
     }
